specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/examples/WafListAllPolicies.json

```diff
@@ -20,7 +20,7 @@
               "resourceState": "Enabled",
               "provisioningState": "Succeeded",
               "policySettings": {
-                "enabledState": "Enabled",
+                "state": "Enabled",
```

Comment:
Shouldn't this response also have `maxRequestBodySizeInKb` and other new props?

Reply:
yes. Added